spec/rails_helper.rb

```diff
@@ -63,7 +63,7 @@
   # The different available types are documented in the features, such as in
   # https://relishapp.com/rspec/rspec-rails/docs
   config.infer_spec_type_from_file_location!
-
+  config.after { Telegram.bot.reset }
```

Comment:
do u need to reset after every each example?

Reply:
Roman is right. Move to "after" inside your spec

Reply:
preferably add some meta-tag filter, there's no need to do this for examples that are not using telegram bot